src/table.ts

```diff
@@ -18,7 +18,7 @@
 
 import * as arrify from 'arrify';
 import * as Big from 'big.js';
-import {ServiceObject, util, SetMetadataResponse, ResponseCallback} from '@google-cloud/common';
+import * as common from '@google-cloud/common';
```

Comment:
Any particular reason for this?

Reply:
Haha, yeah.  The way we appear to be using proxyquire in the tests relies on stubbing `util` specifically.  If I separated it out, things stopped working.  We can revisit this, but with the size of the change I was really just looking to get things healthy again.